src/chrome/content/rules/TheAnarchistLibrary.org.xml

```diff
@@ -13,7 +13,12 @@
 	<target host="ru.theanarchistlibrary.org" />
 	<target host="es.theanarchistlibrary.org" />
 	<target host="sv.theanarchistlibrary.org" />
+	<target host="da.theanarchistlibrary.org" />
+	<target host="fr.theanarchistlibrary.org" />
 	<target host="www.theanarchistlibrary.org" />
+	<target host="www.anarhisticka-biblioteka.org" />
+	<target host="anarchistischebibliothek.org" />
+	<target host="anarhisticka-biblioteka.net" />
```

Comment:
Even if they redirect to the non-www? www.anarchistischebibliothek.org redirects to anarchistischebibliothek.org and anarhisticka-biblioteka.org redirects to www.anarhisticka-biblioteka.org.